.github/actions/build-and-push-octopus/action.yml

```diff
@@ -35,8 +35,18 @@ runs:
       uses: docker/build-push-action@v5
       with:
         context: ./octopus
-        file: ./octopus/dockerfile
+        file: ./octopus/dockerfile.server
         push: true
         tags: |
           ${{ inputs.acr_registry }}/dibbs-cloud/octopusdeploy:${{ env.VERSION }}
-          ${{ inputs.acr_registry }}/dibbs-cloud/octopusdeploy:latest
\ No newline at end of file
+          ${{ inputs.acr_registry }}/dibbs-cloud/octopusdeploy:latest
+
+    - name: Build and push Docker image
```

Comment:
Can we modify this to indicate that it's the tentacle? Or something else to set it apart from the step outlined at line 34?

Reply:
Done.